openshift_scalability/nodeVertical.sh

```diff
@@ -16,17 +17,41 @@ long_sleep() {
 
 clean() { echo "Cleaning environment"; oc delete project clusterproject0; }
 
-# sleeping to gather some steady-state metrics, pre-test
-long_sleep
+golang_clusterloader() {
+  # Export kube config
+  export KUBECONFIG=${KUBECONFIG-$HOME/.kube/config}
+  MY_CONFIG=config/golang/nodeVertical
```

Comment:
Change to nodeVertical.yaml to match file

Reply:
golang clusterloader doesn't require a .yaml extension.

Reply:
That's just strange that the viper config value and filename would not match.

Reply:
I think it assumes that the viper config value has a .yaml extension.